src/main/java/com/library/model/entity/mongo/Book.java

```diff
@@ -0,0 +1,33 @@
+package com.library.model.entity.mongo;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
+import com.library.model.dto.AuthorNameDto;
+import com.library.model.dto.PublisherTitleDto;
+import lombok.Getter;
+import lombok.Setter;
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.MongoId;
+
+import java.util.List;
+
+@Document
+@Getter
+@Setter
+public class Book {
```

Comment:
Not good practice to have the same class names in project.
Code becomes hard to read and understand.

Rename, please.

Reply:
Ok